server/src/test/java/org/candlepin/policy/js/RulesObjectMapperTest.java

```diff
@@ -121,12 +120,13 @@ public void filterTimestampsOffAttributes() {
         context.put("pool", p);
 
         String output = objMapper.toJsonString(context);
+
         // Shouldn't see timestamps:
-        assertFalse(output.contains("\"created\""));
-        assertFalse(output.contains("\"updated\""));
+        assertFalse(output, output.contains("\"created\""));
+        assertFalse(output, output.contains("\"updated\""));
 
         // Shouldn't see a productId:
-        assertFalse(output.contains("PRODID"));
+        assertFalse(output, output.contains("PRODID"));
```

Comment:
Nitpick: these changes are good, but are unrelated to the changes necessary for Jackson... Separate PR please?